cryosparc/spec.py

```diff
@@ -87,6 +87,7 @@
     "text/html",
     "application/json",
     "application/xml",
+    "application/x-troff",
```

Comment:
This is the content type you get when downloading an example BILD file from the chimera website: https://www.cgl.ucsf.edu/chimera/docs/UsersGuide/bild.html

```sh
❯ curl -I https://www.cgl.ucsf.edu/chimera/docs/UsersGuide/example.bild
HTTP/1.1 200 OK
Date: Wed, 31 Jul 2024 14:38:52 GMT
Server: Apache
Strict-Transport-Security: max-age=63072000; includeSubdomains; preload
Last-Modified: Thu, 02 Sep 2021 23:57:03 GMT
ETag: "22f-5cb0bf07791c0"
Accept-Ranges: bytes
Content-Length: 559
Connection: close
Content-Type: application/x-troff
```